js/app/filter-options.js

```diff
@@ -0,0 +1,23 @@
+$(document).ready(function() {
+    $("#preview-download").click(function(e) {
+        var errorDimensions = [];
+
+        $("li#filter-option").each(function() {
+            var filterDiv = $(this).find("a");
+
+            if (filterDiv.html().includes("Add at least one")) {
```

Comment:
It might be better to check if the button is disabled rather than checking against a string in the template. Thinking that the string could change over time and would cause additional maintenance on this js file.

Reply:
Logic behind when the button is disabled isn't quite right at the moment so will stick to this while we figure out a solution